easybuild/easyconfigs/c/Chemaxon_Marvin/Chemaxon_Marvin-21.14.eb

```diff
@@ -0,0 +1,69 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+# Cannot use *easyblock = 'Rpm'* because the RPM misses some fields:
+# Summary and License fields must be present in package: (main package)
+easyblock = 'Binary'
+
+name = 'Chemaxon_Marvin'
+version = '21.14'
+
+homepage = 'https://chemaxon.com/products/marvin'
+description = """Marvin suite is a chemically intelligent desktop toolkit built
+to help you draw, edit, publish, render, import and export your chemical structures
+and as well as allowing you to convert between various chemical and graphical file
+formats. It is free for individual, academic and non-commercial use."""
+
+# NOTE not in the license list of EB
+# software_license = 'Proprietary: Personal, Commercial or Academic'
+software_license_urls = ['https://www.chemaxon.com/marvin/license.html']
+docurls = [
+    'https://docs.chemaxon.com/display/docs/marvinsketch-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/marvinview-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/file-formats.md',
+]
+
+toolchain = SYSTEM
+
+# Download and license at https://chemaxon.com/products/marvin
```

Comment:
I think a bit more detail is required here, given that I have registered for an account and I can't immediately see a link that matches the filename listed below.

Reply:
They changed the web site and download accesses regularly.
Do you want I add this link to help to find the RPM?
https://chemaxon.com/products/marvin/download#download

I will have to change (again) things for the next versions...